gradle.properties

```diff
@@ -1,6 +1,6 @@
 GROUP_ID=com.xing.api
-VERSION=3.0.5
-VERSION_CODE=14
+VERSION=3.0.6
+VERSION_CODE=1
```

Comment:
I'm not sure, what the usage/meaning of `version_code`. Perhaps, it should increase all the time?

Reply:
As I understand it's the build number of that version. In that case it would be correct to reset it per version bump.

Reply:
Reading some [documentation](https://developer.android.com/studio/publish/versioning) I think you are right @MyDogTom . 

> This number is used only to determine whether one version is more recent than another, with higher numbers indicating more recent versions

I will set it to `15`